src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -53,6 +62,131 @@ struct CPU {
         if (_cores == 0) {
             _cores = _processors;
         }
+
+        /**
+         * New method to get CPU infomation and CPU map. Below is the structure of CPU map and two sample.
+         *  1. Four processors of two Pcore
+         *  2. Four processors of four Ecores shared L2 cache
```

Comment:
To be honest the description is not clear. It it the case than we have two Big cores with enabled hyperthreads + 4 Little cores?

Reply:
updated in new commit